Makefile

```diff
@@ -117,11 +123,18 @@ ifneq ($(SKIP_TESTS), true)
 endif
 
 .PHONY: test-scorecard
-test-scorecard: destroy_cryostat_cr undeploy uninstall
+test-scorecard: clean-scorecard
 ifneq ($(SKIP_TESTS), true)
-	operator-sdk scorecard bundle
+	$(CLUSTER_CLIENT) create namespace $(SCORECARD_NAMESPACE)
+	$(CLUSTER_CLIENT) -n $(SCORECARD_NAMESPACE) create -f internal/images/custom-scorecard-tests/rbac/
+	operator-sdk run bundle -n $(SCORECARD_NAMESPACE) $(BUNDLE_IMG)
+	operator-sdk scorecard -n $(SCORECARD_NAMESPACE) -s cryostat-scorecard -w 5m $(BUNDLE_IMG)
 endif
 
+.PHONY: clean-scorecard
```

Comment:
Our custom scorecard test resources appear to persist after tests. Should we remove them after tests like other scorecards?

Reply:
Sure, I added a delete command after the tests run. Also included the `--ignore-not-found` argument used elsewhere.

Reply:
Great!